editor/src/clj/editor/bundle_dialog.clj

```diff
@@ -250,41 +250,46 @@
     (ui/value! variant-choice-box (prefs/get-prefs prefs "bundle-variant" "debug"))
     (ui/value! compression-choice-box (prefs/get-prefs prefs "bundle-texture-compression" "enabled"))
     (ui/value! generate-debug-symbols-check-box (prefs/get-prefs prefs "bundle-generate-debug-symbols?" true))
     (ui/value! generate-build-report-check-box (prefs/get-prefs prefs "bundle-generate-build-report?" false))
-    (ui/value! publish-live-update-content-check-box (prefs/get-prefs prefs "bundle-publish-live-update-content?" false))))
+    (ui/value! publish-live-update-content-check-box (prefs/get-prefs prefs "bundle-publish-live-update-content?" false))
+    (ui/value! bundle-contentless-check-box (prefs/get-prefs prefs "bundle-contentless?" false))))
 
 (defn- save-generic-prefs! [prefs view]
-  (ui/with-controls view [variant-choice-box compression-choice-box generate-debug-symbols-check-box generate-build-report-check-box publish-live-update-content-check-box]
+  (ui/with-controls view [variant-choice-box compression-choice-box generate-debug-symbols-check-box generate-build-report-check-box publish-live-update-content-check-box bundle-contentless-check-box]
     (prefs/set-prefs prefs "bundle-variant" (ui/value variant-choice-box))
     (prefs/set-prefs prefs "bundle-texture-compression" (ui/value compression-choice-box))
     (prefs/set-prefs prefs "bundle-generate-debug-symbols?" (ui/value generate-debug-symbols-check-box))
     (prefs/set-prefs prefs "bundle-generate-build-report?" (ui/value generate-build-report-check-box))
-    (prefs/set-prefs prefs "bundle-publish-live-update-content?" (ui/value publish-live-update-content-check-box))))
+    (prefs/set-prefs prefs "bundle-publish-live-update-content?" (ui/value publish-live-update-content-check-box))
+    (prefs/set-prefs prefs "bundle-contentless?" (ui/value bundle-contentless-check-box))))
 
 (defn- get-generic-options [view]
-  (ui/with-controls view [variant-choice-box compression-choice-box generate-debug-symbols-check-box generate-build-report-check-box publish-live-update-content-check-box]
+  (ui/with-controls view [variant-choice-box compression-choice-box generate-debug-symbols-check-box generate-build-report-check-box publish-live-update-content-check-box bundle-contentless-check-box]
     {:variant (ui/value variant-choice-box)
      :texture-compression (ui/value compression-choice-box)
      :generate-debug-symbols? (ui/value generate-debug-symbols-check-box)
      :generate-build-report? (ui/value generate-build-report-check-box)
      :publish-live-update-content? (and (ui/value publish-live-update-content-check-box)
-                                        (ui/editable publish-live-update-content-check-box))}))
+                                        (ui/editable publish-live-update-content-check-box))
+     :ignore-resource-archive? (ui/value bundle-contentless-check-box)}))
```

Comment:
Shouldn't this be renamed to `:bundle-contentless?` as well? Everything else is consistent with their preference key and check-box variable names.

Reply:
i missed that ( 
thank you